src/main/java/com/porotov/articleservice/model/Article.java

```diff
@@ -0,0 +1,51 @@
+package com.porotov.articleservice.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.Hibernate;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+/**@author Alexandr Porotov
+ * @version  pre-alpha
+ * */
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Entity(name = "Article")
+@Table(name = "articles", schema = "public")
+public class Article {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "article_id")
+    private Long id;
+
+    @Column(name = "article_title")
+    private String title;
+
+    @Column(name = "article_url")
+    private String url;
+
+    @Column(name = "article_creation_date")
+    private LocalDateTime dateCreationTime = LocalDateTime.now();
+
+    @Column(name = "article_change_date")
+    private LocalDateTime dateChangeTime;
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
@Data has @equals and hashcode, if you want to use some standart override with your fields. If you want to exclude field from method, just use ToString.Exclude, etc.